### ISSUE TITLE
Application fails to run after rebuilding with apktool

### ISSUE THREAD

**Issue by sirnaysayer** [iBotPeaches/Apktool#1715]
### Information
1. **Apktool Version (`apktool -version`)** - 2.3.1
2. **Operating System (Mac, Linux, Windows)** - Linux
3. **APK From? (Playstore, ROM, Other)** - Playstore

### Stacktrace/Logcat

```
                            Process com.grindrapp.android created for activity com.grindrapp.android/.SplashActivity-Original
                            PID: 21261   UID: 10105   GIDs: {50105, 3003, 1028, 1015}

               dalvikvm  D  GC_CONCURRENT freed 216K, 10% free 3143K/3460K, paused 2ms+1ms, total 21ms
                         D  DexOpt: --- BEGIN 'com.grindrapp.android-1.apk.classes2.zip' (bootstrap=0) ---
                         D  DexOpt: --- END 'com.grindrapp.android-1.apk.classes2.zip' (success) ---
                         D  DEX prep '/data/data/com.grindrapp.android/code_cache/secondary-dexes/com.grindrapp.android-1.apk.classes2.zip': unzip in 780ms, rewrite 3878ms
                         I  Could not find method android.content.Context.createDeviceProtectedStorageContext, referenced from method android.support.v4.content.ContextCompat.createDeviceProtectedStorageContext
                         W  VFY: unable to resolve virtual method 552: Landroid/content/Context;.createDeviceProtectedStorageContext ()Landroid/content/Context;
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.getCodeCacheDir, referenced from method android.support.v4.content.ContextCompat.getCodeCacheDir
                         W  VFY: unable to resolve virtual method 561: Landroid/content/Context;.getCodeCacheDir ()Ljava/io/File;
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.getColor, referenced from method android.support.v4.content.ContextCompat.getColor
                         W  VFY: unable to resolve virtual method 562: Landroid/content/Context;.getColor (I)I
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.getColorStateList, referenced from method android.support.v4.content.ContextCompat.getColorStateList
                         W  VFY: unable to resolve virtual method 563: Landroid/content/Context;.getColorStateList (I)Landroid/content/res/ColorStateList;
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.getDataDir, referenced from method android.support.v4.content.ContextCompat.getDataDir
                         W  VFY: unable to resolve virtual method 565: Landroid/content/Context;.getDataDir ()Ljava/io/File;
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.getDrawable, referenced from method android.support.v4.content.ContextCompat.getDrawable
                         W  VFY: unable to resolve virtual method 568: Landroid/content/Context;.getDrawable (I)Landroid/graphics/drawable/Drawable;
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.getNoBackupFilesDir, referenced from method android.support.v4.content.ContextCompat.getNoBackupFilesDir
                         W  VFY: unable to resolve virtual method 576: Landroid/content/Context;.getNoBackupFilesDir ()Ljava/io/File;
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.isDeviceProtectedStorage, referenced from method android.support.v4.content.ContextCompat.isDeviceProtectedStorage
                         W  VFY: unable to resolve virtual method 591: Landroid/content/Context;.isDeviceProtectedStorage ()Z
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.startForegroundService, referenced from method android.support.v4.content.ContextCompat.startForegroundService
                         W  VFY: unable to resolve virtual method 610: Landroid/content/Context;.startForegroundService (Landroid/content/Intent;)Landroid/content/ComponentName;
                         D  VFY: replacing opcode 0x6e at 0x0006
                         I  Could not find method android.content.Context.getNoBackupFilesDir, referenced from method com.google.android.gms.common.util.zzw.getNoBackupFilesDir
                         W  VFY: unable to resolve virtual method 576: Landroid/content/Context;.getNoBackupFilesDir ()Ljava/io/File;
                         D  VFY: replacing opcode 0x6e at 0x0006
                         D  GC_CONCURRENT freed 243K, 10% free 3297K/3624K, paused 2ms+6ms, total 19ms
                         I  Could not find method android.content.pm.PackageManager.getPackageInstaller, referenced from method com.google.android.gms.common.zze.zzs
                         W  VFY: unable to resolve virtual method 751: Landroid/content/pm/PackageManager;.getPackageInstaller ()Landroid/content/pm/PackageInstaller;
                         D  VFY: replacing opcode 0x6e at 0x0010
                     FA  I  App measurement is starting up, version: 10084
                         I  To enable debug logging run: adb shell setprop log.tag.FA VERBOSE
                         I  To enable faster debug mode event logging run:
                         I    adb shell setprop debug.firebase.analytics.app com.grindrapp.android
               dalvikvm  D  GC_CONCURRENT freed 633K, 19% free 3141K/3836K, paused 2ms+2ms, total 23ms
                         D  GC_CONCURRENT freed 334K, 17% free 3216K/3836K, paused 2ms+1ms, total 13ms
                         W  VFY: unable to find class referenced in signature (Lrx/Observable;)
                         D  GC_CONCURRENT freed 251K, 12% free 3385K/3836K, paused 2ms+1ms, total 15ms
                         D  GC_CONCURRENT freed 221K, 8% free 3674K/3976K, paused 3ms+2ms, total 26ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 7ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 7ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 7ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 8ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 9ms
                         D  GC_CONCURRENT freed 332K, 10% free 3847K/4268K, paused 2ms+1ms, total 19ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 12ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 6ms
                         I  Could not find method android.content.Context.getNoBackupFilesDir, referenced from method com.localytics.android.MarketingHandler._createLocalyticsDirectory
                         W  VFY: unable to resolve virtual method 351: Landroid/content/Context;.getNoBackupFilesDir ()Ljava/io/File;
                         D  VFY: replacing opcode 0x6e at 0x000d
                         I  Could not find method android.content.Context.getNoBackupFilesDir, referenced from method com.localytics.android.BaseProvider.newDatabaseDir
                         W  VFY: unable to resolve virtual method 351: Landroid/content/Context;.getNoBackupFilesDir ()Ljava/io/File;
                         D  VFY: replacing opcode 0x6e at 0x0002
                         I  Could not find method android.content.Context.getNoBackupFilesDir, referenced from method com.localytics.android.CreativeManager.getZipFileDirPath
                         W  VFY: unable to resolve virtual method 576: Landroid/content/Context;.getNoBackupFilesDir ()Ljava/io/File;
                         D  VFY: replacing opcode 0x6e at 0x0014
                         I  Could not find method android.content.Context.getNoBackupFilesDir, referenced from method com.localytics.android.MarketingProvider.determineCreativeLocation
                         W  VFY: unable to resolve virtual method 351: Landroid/content/Context;.getNoBackupFilesDir ()Ljava/io/File;
                         D  VFY: replacing opcode 0x6e at 0x0010
                         D  GC_CONCURRENT freed 349K, 10% free 3905K/4336K, paused 3ms+7ms, total 26ms
                         D  GC_FOR_ALLOC freed 475K, 16% free 3783K/4492K, paused 14ms, total 14ms
                         D  GC_FOR_ALLOC freed 13K, 16% free 3807K/4492K, paused 13ms, total 13ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.361MB for 70704-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 49K, 16% free 3849K/4564K, paused 13ms, total 13ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.402MB for 70704-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 16% free 3918K/4636K, paused 13ms, total 13ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.469MB for 70704-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 69K, 17% free 3924K/4708K, paused 13ms, total 13ms
                         D  GC_FOR_ALLOC freed 1K, 16% free 3980K/4708K, paused 13ms, total 13ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.518MB for 57864-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 1K, 16% free 4038K/4768K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.575MB for 57864-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 56K, 17% free 4038K/4828K, paused 13ms, total 13ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.578MB for 61288-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 17% free 4098K/4888K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.636MB for 61288-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 16% free 4161K/4948K, paused 13ms, total 13ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.698MB for 61288-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 60K, 16% free 4218K/5008K, paused 14ms, total 14ms
                         D  GC_FOR_ALLOC freed 2K, 15% free 4277K/5008K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.806MB for 56240-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 7K, 15% free 4327K/5064K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.854MB for 55416-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 1K, 15% free 4379K/5120K, paused 12ms, total 12ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.906MB for 55416-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 56K, 16% free 4380K/5176K, paused 18ms, total 18ms
                         D  GC_FOR_ALLOC freed 1K, 15% free 4434K/5176K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.944MB for 39984-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 15% free 4474K/5216K, paused 13ms, total 13ms
          dalvikvm-heap  I  Grow heap (frag case) to 4.983MB for 39984-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 54K, 16% free 4459K/5256K, paused 14ms, total 14ms
                         D  GC_FOR_ALLOC freed 40K, 16% free 4460K/5256K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.004MB for 76240-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 15% free 4535K/5332K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.077MB for 76240-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 5K, 15% free 4615K/5408K, paused 15ms, total 15ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.156MB for 76240-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 14% free 4744K/5484K, paused 19ms, total 19ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.262MB for 55656-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 83K, 15% free 4729K/5540K, paused 14ms, total 14ms
                         D  GC_FOR_ALLOC freed <1K, 14% free 4783K/5540K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.325MB for 80944-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 14% free 4863K/5620K, paused 12ms, total 12ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.402MB for 80944-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 56K, 15% free 4888K/5700K, paused 15ms, total 15ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.426MB for 80944-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 15% free 4967K/5780K, paused 16ms, total 16ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.485MB for 60960-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 2K, 14% free 5029K/5840K, paused 16ms, total 16ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.545MB for 60960-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed 82K, 16% free 5011K/5900K, paused 14ms, total 14ms
                         D  GC_FOR_ALLOC freed 81K, 15% free 5049K/5900K, paused 14ms, total 14ms
                         D  GC_FOR_ALLOC freed <1K, 14% free 5106K/5900K, paused 15ms, total 15ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.618MB for 57552-byte allocation
               dalvikvm  D  GC_FOR_ALLOC freed <1K, 14% free 5163K/5960K, paused 14ms, total 14ms
          dalvikvm-heap  I  Grow heap (frag case) to 5.672MB for 57552-byte allocation
               dalvikvm  D  DexOpt: --- BEGIN 'generated-8CB83149C9F111667818651F0A43B62FD1A557A0.jar' (bootstrap=0) ---
         ActivityThread  E  Failed to find provider info for com.facebook.katana.provider.AttributionIdProvider
               dalvikvm  D  DexOpt: --- END 'generated-8CB83149C9F111667818651F0A43B62FD1A557A0.jar' (success) ---
                         D  DEX prep '/data/data/com.grindrapp.android/cache/generated-8CB83149C9F111667818651F0A43B62FD1A557A0.jar': unzip in 0ms, rewrite 156ms
                         D  Trying to load lib /data/data/com.grindrapp.android/files/libcrashreport.so 0x4190c0f0
                         D  Added shared lib /data/data/com.grindrapp.android/files/libcrashreport.so 0x4190c0f0
                         D  No JNI_OnLoad found in /data/data/com.grindrapp.android/files/libcrashreport.so 0x4190c0f0, skipping init
                         D  GC_FOR_ALLOC freed 445K, 13% free 5285K/6020K, paused 17ms, total 18ms
                         D  GC_CONCURRENT freed 432K, 9% free 5605K/6112K, paused 9ms+3ms, total 41ms
                         D  GC_CONCURRENT freed 673K, 12% free 5832K/6584K, paused 7ms+5ms, total 42ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 8ms
         ActivityThread  E  Failed to find provider info for com.facebook.katana.provider.AttributionIdProvider
               dalvikvm  I  Could not find method android.view.Window$Callback.onProvideKeyboardShortcuts, referenced from method android.support.v7.view.WindowCallbackWrapper.onProvideKeyboardShortcuts
                         W  VFY: unable to resolve interface method 17679: Landroid/view/Window$Callback;.onProvideKeyboardShortcuts (Ljava/util/List;Landroid/view/Menu;I)V
                         D  VFY: replacing opcode 0x72 at 0x0002
                         W  VFY: unable to find class referenced in signature (Landroid/view/SearchEvent;)
                         I  Could not find method android.view.Window$Callback.onSearchRequested, referenced from method android.support.v7.view.WindowCallbackWrapper.onSearchRequested
                         W  VFY: unable to resolve interface method 17681: Landroid/view/Window$Callback;.onSearchRequested (Landroid/view/SearchEvent;)Z
                         D  VFY: replacing opcode 0x72 at 0x0002
                         I  Could not find method android.view.Window$Callback.onWindowStartingActionMode, referenced from method android.support.v7.view.WindowCallbackWrapper.onWindowStartingActionMode
                         W  VFY: unable to resolve interface method 17685: Landroid/view/Window$Callback;.onWindowStartingActionMode (Landroid/view/ActionMode$Callback;I)Landroid/view/ActionMode;
                         D  VFY: replacing opcode 0x72 at 0x0002
                         I  Could not find method android.content.res.TypedArray.getChangingConfigurations, referenced from method android.support.v7.widget.TintTypedArray.getChangingConfigurations
                         W  VFY: unable to resolve virtual method 852: Landroid/content/res/TypedArray;.getChangingConfigurations ()I
                         D  VFY: replacing opcode 0x6e at 0x0002
                         I  Could not find method android.content.res.TypedArray.getType, referenced from method android.support.v7.widget.TintTypedArray.getType
                         W  VFY: unable to resolve virtual method 874: Landroid/content/res/TypedArray;.getType (I)I
                         D  VFY: replacing opcode 0x6e at 0x0008
                         D  GC_CONCURRENT freed 762K, 12% free 6032K/6828K, paused 3ms+20ms, total 50ms
                         D  DexOpt: couldn't find field Landroid/os/Message;.sendingUid
                         W  VFY: unable to resolve instance field 216
                         D  VFY: replacing opcode 0x52 at 0x0000
                         D  GC_CONCURRENT freed 827K, 13% free 6261K/7168K, paused 35ms+3ms, total 98ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 30ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 36ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 27ms
                         D  GC_CONCURRENT freed 921K, 14% free 6471K/7472K, paused 3ms+10ms, total 105ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 79ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 32ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 82ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 75ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 85ms
                         D  GC_CONCURRENT freed 805K, 12% free 6867K/7752K, paused 9ms+3ms, total 69ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 29ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 23ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 14ms
                         D  GC_CONCURRENT freed 1515K, 20% free 6685K/8280K, paused 4ms+9ms, total 79ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 49ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 52ms
                         D  GC_CONCURRENT freed 1448K, 23% free 6400K/8280K, paused 4ms+5ms, total 51ms
                         W  VFY: unable to find class referenced in signature (Ljava/nio/file/Path;)
                         W  VFY: unable to find class referenced in signature ([Ljava/nio/file/OpenOption;)
                         I  Could not find method java.nio.file.Files.newOutputStream, referenced from method okio.Okio.sink
                         W  VFY: unable to resolve static method 29612: Ljava/nio/file/Files;.newOutputStream (Ljava/nio/file/Path;[Ljava/nio/file/OpenOption;)Ljava/io/OutputStream;
                         D  VFY: replacing opcode 0x71 at 0x000a
                         W  VFY: unable to find class referenced in signature (Ljava/nio/file/Path;)
                         W  VFY: unable to find class referenced in signature ([Ljava/nio/file/OpenOption;)
                         I  Could not find method java.nio.file.Files.newInputStream, referenced from method okio.Okio.source
                         W  VFY: unable to resolve static method 29611: Ljava/nio/file/Files;.newInputStream (Ljava/nio/file/Path;[Ljava/nio/file/OpenOption;)Ljava/io/InputStream;
                         D  VFY: replacing opcode 0x71 at 0x000a
                         D  GC_CONCURRENT freed 966K, 21% free 6610K/8280K, paused 2ms+3ms, total 71ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 31ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 33ms
                         I  Total arena pages for JIT: 11
                         I  Total arena pages for JIT: 12
                         I  Total arena pages for JIT: 13
                         I  Total arena pages for JIT: 14
         InstanceID/Rpc  W  Found 10013
               dalvikvm  D  GC_CONCURRENT freed 1354K, 22% free 6502K/8280K, paused 4ms+3ms, total 40ms
                         D  WAIT_FOR_CONCURRENT_GC blocked 13ms
                         D  GC_CONCURRENT freed 1151K, 21% free 6556K/8280K, paused 7ms+3ms, total 33ms
                         I  Jit: resizing JitTable from 4096 to 8192
                         D  GC_CONCURRENT freed 1110K, 21% free 6579K/8280K, paused 4ms+11ms, total 55ms
         ActivityThread  E  Failed to find provider info for com.facebook.katana.provider.AttributionIdProvider
                         I  Could not find method android.view.Window$Callback.onProvideKeyboardShortcuts, referenced from method android.support.v7.view.WindowCallbackWrapper.onProvideKeyboardShortcuts
                         W  VFY: unable to resolve interface method 17679: Landroid/view/Window$Callback;.onProvideKeyboardShortcuts (Ljava/util/List;Landroid/view/Menu;I)V
                         D  VFY: replacing opcode 0x72 at 0x0002
                         W  VFY: unable to find class referenced in signature (Landroid/view/SearchEvent;)
                         I  Could not find method android.view.Window$Callback.onSearchRequested, referenced from method android.support.v7.view.WindowCallbackWrapper.onSearchRequested
                         W  VFY: unable to resolve interface method 17681: Landroid/view/Window$Callback;.onSearchRequested (Landroid/view/SearchEvent;)Z
                         D  VFY: replacing opcode 0x72 at 0x0002
                         I  Could not find method android.view.Window$Callback.onWindowStartingActionMode, referenced from method android.support.v7.view.WindowCallbackWrapper.onWindowStartingActionMode
                         W  VFY: unable to resolve interface method 17685: Landroid/view/Window$Callback;.onWindowStartingActionMode (Landroid/view/ActionMode$Callback;I)Landroid/view/ActionMode;
                         D  VFY: replacing opcode 0x72 at 0x0002
                         I  Could not find method android.content.res.TypedArray.getChangingConfigurations, referenced from method android.support.v7.widget.TintTypedArray.getChangingConfigurations
                         W  VFY: unable to resolve virtual method 852: Landroid/content/res/TypedArray;.getChangingConfigurations ()I
                         D  VFY: replacing opcode 0x6e at 0x0002
                         I  Could not find method android.content.res.TypedArray.getType, referenced from method android.support.v7.widget.TintTypedArray.getType
                         W  VFY: unable to resolve virtual method 874: Landroid/content/res/TypedArray;.getType (I)I
```

### Steps to Reproduce
1. `java -jar apktool.jar d -r com.grindrapp.android-1.apk -o grindr/ `
2. `java -jar apktool.jar b -r grindr/ -o com.grindrapp.android-1-test.apk`
3. `java -jar uber-apk-signer.jar -a com.grindrapp.android-1-test.apk --out grindr-signed`
4. `adb install grindr-signed/com.grindrapp.android-1-test-aligned-debugSigned.apk`

### Frameworks
If this APK is from an OEM ROM (Samsung, HTC, LG). Please attach framework files
(`.apks` that live in `/system/framework` or `/system/priv-app`)

N/A

### APK
If this APK can be freely shared, please upload/attach a link to it.

https://apkpure.com/grindr-gay-chat-meet-date/com.grindrapp.android/download?from=details

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? - Yes, some minor resource errors.
```
I: Using Apktool 2.3.1
I: Checking whether sources has changed...
I: Checking whether sources has changed...
I: Checking whether resources has changed...
I: Building resources...
S: WARNING: Could not write to (/root/.local/share/apktool/framework), using /tmp instead...
S: Please be aware this is a volatile directory and frameworks could go missing, please utilize --frame-path if the default storage directory is unavailable
W: /root/grindr/AndroidManifest.xml:93: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:95: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:98: Tag <activity> attribute name has invalid character 'Z'. Ignoring...
W: /root/grindr/AndroidManifest.xml:99: Tag <activity> attribute name has invalid character ''. Ignoring...
W: /root/grindr/AndroidManifest.xml:102: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:103: Tag <activity> attribute name has invalid character 'q'. Ignoring...
W: /root/grindr/AndroidManifest.xml:104: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:117: Tag <activity> attribute name has invalid character '*'. Ignoring...
W: /root/grindr/AndroidManifest.xml:118: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:119: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:120: Tag <activity> attribute name has invalid character '0'. Ignoring...
W: /root/grindr/AndroidManifest.xml:121: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:122: Tag <activity> attribute name has invalid character 'b'. Ignoring...
W: /root/grindr/AndroidManifest.xml:123: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:124: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:159: Tag <activity> attribute name has invalid character '*'. Ignoring...
W: /root/grindr/AndroidManifest.xml:160: Tag <activity> attribute name has invalid character ''. Ignoring...
W: /root/grindr/AndroidManifest.xml:172: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:174: Tag <activity> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:196: Tag <provider> attribute name has invalid character '�'. Ignoring...
W: /root/grindr/AndroidManifest.xml:204: Tag <receiver> attribute name has invalid character '='. Ignoring...
I: Building apk file...
I: Copying unknown files/dir...
```
2. If you are trying to install a modified apk, did you resign it? - Yes, both manually and with uber-apk-signer, both using a debug key.
3. Are you using the latest apktool version? Yes, 2.3.1

Decoding and building finish with no errors or warnings with the -r flag, app does not respond when running on real hardware or bluestacks. I'm not seeing the issue in logcat. The application seems to be running in the background but the window instantly closes when brought to foreground.

**Comment by MouadSb** [comment]
i've the same issue

**Comment by iBotPeaches** [comment]
The manifest garbled attributes will be fixed in v2.5.0 and I'm sorry for 2-3 year delay on response on first error. I don't know and with how many changes in Apktool since then, I'm just going to close as too much has changed.
